### PR TITLE
Update upcoming_events.yml

### PR DESCRIPTION
**Pull request by cygov**

### CODE REVIEW

File: _content/upcoming_events.yml [hunk 1]
```diff
@@ -5,11 +5,26 @@
...
+- name: "Unbox"
+  url: "https://bosproduct.com/unbox/unbox2019/"
+  date: September 20
+  city: Boston, MA
+  venue: Marriott Copley Boston - 110 Huntington Ave.
+  summary: "Unbox is a day to elevate PMs on the front lines of work. The U.S. Digital Service is recruiting the nation's top product managers to serve tours of duty in government. Help us build products that change the way Americans interact with their government."
+  image_name:  
+  
+  - name: "Grace Hopper Celebration"
+  url: "https://lesbianswhotech.org/newyork2019/"
```
Comment (jaredcunha-usds) [inline _content/upcoming_events.yml]: @cygov - Looks like the website for the Grace Hopper celebration is the same as Lesbians Who Tech Leadership Summit. I didn't dig too deeply into it, but is that correct?

File: _content/upcoming_events.yml [hunk 2]
```diff
@@ -5,11 +5,26 @@
...
+
+- name: "Unbox"
+  url: "https://bosproduct.com/unbox/unbox2019/"
+  date: September 20
+  city: Boston, MA
+  venue: Marriott Copley Boston - 110 Huntington Ave.
+  summary: "Unbox is a day to elevate PMs on the front lines of work. The U.S. Digital Service is recruiting the nation's top product managers to serve tours of duty in government. Help us build products that change the way Americans interact with their government."
+  image_name:
+
+- name: "Grace Hopper Celebration"
```
Comment (jaredcunha-usds) [inline _content/upcoming_events.yml:24]: There was an incorrect indent here, which I fixed.

**Comment by jaredcunha-usds** [comment]
@cygov Following up on this. Did you see the comment on the link?

**Comment by cygov** [comment]
Fixed @jaredcunha, thank you!

**Review by jaredcunha-usds — APPROVED** [review]

**Pull request merged**